src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,105 @@
+package homework_2.random_chars_table;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Random;
+
+class RandomCharsTable {
+
+    private int length;
+    private int width;
+    private boolean isEven;
+
+    void run() {
+
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+
+            if (valid(reader.readLine())) {
+                printTable();
+            }
+
+        } catch (IOException e) {
+            System.out.println("Error: " + e.getMessage());
+        }
+    }
+
+    private void printTable() {
+
+        char[][] arr = new char[length][width];
+        Random r = new Random();
+        StringBuilder result = new StringBuilder();
+        StringBuilder massive = new StringBuilder();
+
+        for (int i = 0; i < arr.length; i++) {
+
+            boolean hasNext = false;
```

Comment:
will it work without hasNext?

Reply:
Yes, but there will be unnesasary chars. I think it could be more efective way to solve it.